src/rules/sortImportsRule.ts

```diff
@@ -0,0 +1,242 @@
+'use strict';
+
+import * as Lint from 'tslint';
+import * as ts from 'typescript';
+
+const RULE_NAME = 'sort-imports';
+
+export class Rule extends Lint.Rules.AbstractRule {
+  public static metadata: Lint.IRuleMetadata = {
+    ruleName: RULE_NAME,
+    description: 'enforce sorting import declarations within module',
+    descriptionDetails: Lint.Utils.dedent`
```

Comment:
Use rationale instead of descriptionDetails since this is what we use to generate the documentation. See https://github.com/buzinas/tslint-eslint-rules/blob/master/src/rules/terArrowBodyStyleRule.ts as an example.